utils/utils_unix_test.go

```diff
@@ -1,6 +1,11 @@
+//go:build unix
```

Comment:
Can you remove those build tags also?

Reply:
Not like "_GOOS.go", "_unix.go" is just a name. It will have no effect in windows platform.
And you'll get some compile-time error like "TestXxx redeclared in this block" in the windows platform.
So the build tag can't be removed.

Reply:
I made a demo for it. https://github.com/kvii/unix-is-not-goos